swarm/swarm.go

```diff
@@ -109,24 +104,24 @@ func NewSwarm(ctx *node.ServiceContext, backend chequebook.Backend, config *api.
 		return nil, fmt.Errorf("empty bzz key")
 	}
 
+	var backend chequebook.Backend
+	if config.SwapApi != "" && config.SwapEnabled {
+		log.Info("connecting to SWAP API", "url", config.SwapApi)
+		backend, err = ethclient.Dial(config.SwapApi)
+		if err != nil {
+			return nil, fmt.Errorf("error connecting to SWAP API %s: %s", config.SwapApi, err)
+		}
+	}
+
```

Comment:
was there a reason why this was initialized outside before?

Reply:
To have ipcEndpoint available in all test cases.

Reply:
and I guess now we don't need that anymore?